internal/cmdutils/pflag.go

```diff
@@ -154,6 +154,14 @@ func AddInteractiveFlag(cmd *cobra.Command) func() {
 	}
 }
 
+func AddOutputPathFlag(cmd *cobra.Command) func() {
+	cmd.Flags().StringP("output", "o", "",
+		"Optional path of the created bundle artifact or coverage report.")
```

Comment:
```suggestion
		"Optional path of the file(s) created by this command  (eg. coverage report, bundle artifact, ...)
```
wdyt? This way we don't have to change the message when using it in another command.